central/complianceoperator/v2/benchmarks/store/postgres/gen.go

```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.ComplianceOperatorBenchmarkV2 --search-category COMPLIANCE_BENCHMARK --feature-flag ComplianceEnhancements
```

Comment:
Need to add a reference to `storage.ComplianceOperatorProfileV2` so that the search framework will be able to find what it needs

Reply:
Done